src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -570,6 +577,81 @@ def disable_freeu(self):
         """Disables the FreeU mechanism if enabled."""
         self.unet.disable_freeu()
 
+    @property
+    def guidance_scale(self):
+        return self._guidance_scale
+
+    @property
+    def guidance_rescale(self):
+        return self._guidance_rescale
+
+    @property
+    def clip_skip(self):
+        return self._clip_skip
+
+    # here `guidance_scale` is defined analog to the guidance weight `w` of equation (2)
+    # of the Imagen paper: https://arxiv.org/pdf/2205.11487.pdf . `guidance_scale = 1`
+    # corresponds to doing no classifier free guidance.
+    @property
+    def do_classifier_free_guidance(self):
+        return self._guidance_scale > 1
+
+    @property
+    def generator(self):
+        return self._generator
+
+    @property
+    def eta(self):
+        return self._eta
+
+    @property
+    def cross_attention_kwargs(self):
+        return self._cross_attention_kwargs
+
+    @property
+    def extra_step_kwargs(self):
+        return self._extra_step_kwargs
+
+    @property
+    def step_index(self):
+        return self._step_index
+
+    @property
+    def timestep(self):
+        return self._timestep
+
+    @property
+    def timesteps(self):
+        return self._timesteps
+
+    @property
+    def height(self):
+        return self._height
+
+    @property
+    def width(self):
+        return self._width
+
```

Comment:
```suggestion
```

This cannot be adapted during the inference process, let's remove it as well for now